openwisp_monitoring/check/classes/iperf3.py

```diff
@@ -201,6 +201,15 @@ def check(self, store=True):
                 )
             )
             return
+        # Avoid running the iperf3 check when the device monitoring status is "critical"
+        if (
+            self.related_object.monitoring
+            and self.related_object.monitoring.status in ('critical',)
+        ):
+            logger.warning(
+                f'"{self.related_object}" DeviceMonitoring health status is "critical", iperf3 check skipped!'
```

Comment:
let's break this line please as it's a bit too long (let's try to keep text lines compact)